### DISCUSSION TITLE
Subtract expenses from project budget

### DISCUSSION THREAD

**Discussion by mcfly3001** [kimai/kimai#4021] [Questions & Answers]
In my projects I often have to send monthly invoices with a fixed amount. This is independant from how many hours were spent during the month. For example, the customer orders 1.200h for this year with the total costs of 120.000€. The invoicing will be done on a monthly basis, i.e. each month 10.000€.
Hence, the progress of the worked hours and the invoiced budget can differ throughout the year. It's only important that by end of the project, the ordered hours were delivered.

I wanted to do it as follows in Kimai:
- Add activities for Development, Meetings, Others as billable activities but with an hourly rate of 0€. This way, the hourly progress can be tracked.
- Each month, add an expense entry worth 10.000€ for our service in the last month. I was expecting to see an increase of the used budget as soon as I would mark the expense as paid. This way, I could easily update the customer on how much budget was already spent compared to the actual work hours we delivered.

My second assumption though does not work, because the expenses are not subtracted from the overall budget. What is the reason for this? Or is there a better way to do that?

Thanks for any hints!

**Comment by kevinpapst** [comment]
Please re-post here: https://github.com/kevinpapst/kimai-plugins

**Reply by kevinpapst** [reply]
Follow-up at https://github.com/kevinpapst/kimai-plugins/discussions/69
